pkg/apis/core/validation/seed.go

```diff
@@ -356,3 +359,17 @@ func validateSeedOperation(operation string, fldPath *field.Path) field.ErrorLis
 
 	return allErrs
 }
+
+func validateSeedOperationUpdate(newOperation, oldOperation string, fldPath *field.Path) field.ErrorList {
```

Comment:
Do we really need this? We cannot really prevent that somebody removes the annotation either, so is it really helpful to prevent changing it?

Reply:
Without this check the e2e failed https://prow.gardener.cloud/view/gs/gardener-prow/pr-logs/pull/gardener_gardener/8396/pull-gardener-e2e-kind/1697166830726025216
This is what might have happened:
1. `renew_kubeconfig` e2e test set the annotation `gardener.cloud/operation=renew-kubeconfig`.
2. While reconciliation from step 1. was still running `gardener_access` e2e test has overwritten the seed's annotation with `gardener.cloud/operation=renew-garden-access-secrets`.
3. The reconciliation from step 1. finished and removed the `gardener.cloud/operation` annotation.
4. The seed was reconciled again, but without an annotation.
5. `gardener_access` test failed because the secrets renewal task did never run.

The validation does not prevent anyone from deleting `gardener.cloud/operation` annotation while the reconciliation is still running, but it prevents accidents like in the failing test.